update_prevalence.py

```diff
@@ -1604,10 +1604,27 @@ def main() -> None:
     for country in sorted(data.countries.values(), key=namegetter):
         app_locations[country.app_key] = country.as_app_data()
         for state in country.states.values():
-            if state.app_key not in app_locations:
-                app_locations[state.app_key] = state.as_app_data()
-            for county in state.counties.values():
-                app_locations[county.app_key] = county.as_app_data()
+            try:
```

Comment:
Ha! @apiology - now I see the try/catch 👍

Reply:
Hi @coachnate!  This is a different PR than your original comment in https://github.com/microCOVID/microCOVID/pull/1655.  This approach would not have had any effect on https://github.com/microCOVID/microCOVID/issues/1654.